quickstarts/jboss-eap7-with-helm.yaml

```
@@ -0,0 +1,218 @@
+
+    * **Optimized for OpenShift, Cloud and Containers**
+
+    * **Jakarta EE 8 compatibility**
+
+    * **Lightweight, flexible architecture**
+
+    * **Red Hat Portfolio Integration**
+
+    * **More productive developers with DevOps and Agile Development**
+
+    In this quick start, you will deploy and run a [Jakarta EE application](https://github.com/jboss-eap-up-and-running/eap7-getting-started) with JBoss EAP 7 on OpenShift.
+
+    **Note:** The quickstart uses a sample [Jakarta EE application](https://github.com/jboss-eap-up-and-running/eap7-getting-started). You can deploy your own Jakarta EE application using the same steps.
+    The quick start has instructions where changes would be needed (Task 1).
+
+  tasks:
+    - description: >-
+        To create a JBoss EAP 7 application:
+        
+        1. In the main navigation, click the dropdown menu [perspective switcher]{{highlight qs-perspective-switcher}} and select **Developer**.
+        
+        1. In the navigation menu, click [Add]{{highlight qs-nav-add}}.  
+           The **Add** page opens.
+
+        1. In the **Add** page, click **Helm Chart**.
+        
+        1. In the **Helm Charts** catalog, search for **JBoss EAP 7.4**
+        
+        1. Click the **JBoss EAP 7.4**  Helm chart card.
+
+            - The side panel displays information about the JBoss EAP 7 Helm chart
+
+        1. Click **Install Helm Chart**.
+        
+            - Some form sections are collapsed by default. Click **>** to expand and view its content.  
+              **Note:** No updates are required to these sections to proceed.
+
+            - The details about the [Jakarta EE application](https://github.com/jboss-eap-up-and-running/eap7-getting-started) that you are building and deploying are specified in the `build.uri` field:
+
+              ```
+              build:
+                  uri: https://github.com/jboss-eap-up-and-running/eap7-getting-started
+              ```
+
+              **Note:** If you are building a different application, you must change this `uri` field to point to the Git repository of that application.
+
+        1.  Click **Install** to create the JBoss EAP 7 application using the Helm Chart.
+
+            - The console switches to the [Topology]{{highlight qs-nav-topology}} view and displays a note that has details about the Helm release.
+              After you read the note, you can close it.
+      review:
+        failedTaskHelp: This task isn’t verified yet. Try the task again.
+        instructions: |-
+          The Helm release is represented by a dashed box that contains the JBoss EAP icon and **eap74** text. This content is placed outside the dashed box.
+
+          The deployment is indicated by a circle inside the dashed box with text **D eap74**.
```

is the `D` correct ?

```suggestion
          The deployment is indicated by a circle inside the dashed box with text **D eap74**.
```

---

here and below